### PR TITLE
Regenerate the client and fix the build

### PR DESCRIPTION
**Pull request by nyc**
## Changes

Most of the changes in this PR are auto-generated changes as the result of re-running swagger-codegen with a newer version of the tool and a newer version of the proto.  **NOTE:** This is a potentially breaking change for any users of this library as some methods have changed. The explicit changes I made are below:

* Update the CircleCI Python image to 3.8.7 to fix the build.
    * [Link](https://github.com/grafeas/client-python/pull/8/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47L11)
* Add a script and config file for making it easier to regenerate the client in the future.
    * [Link to script](https://github.com/grafeas/client-python/pull/8/files#diff-be3b2769f01dd9f7e4dc733ecacac374f506777d575a92280908410bfd86e89eR1)
    * [Link to config](https://github.com/grafeas/client-python/pull/8/files#diff-e2d6fb9f2eb021de7d6110c9dcfd6882ab20f0624cdae8ce90328c72ff5322d1R1)

## Why
This PR was originally intended to fix the CI build which was failing due to a long-existing issue in setuptools (see https://github.com/pypa/setuptools/issues/498). Since the issue seems to have been resolved by newer versions of setuptools, I updated the Python image version used in the CI pipeline to a newer one. This revealed an issue where the generated client code was using `async` as a keyword param, which is invalid syntax in Python 3.7+ because it is a language keyword. Thus, I regenerated the client code using the newest version of the proto and the current stable version of swagger-codegen v2. Since it took a bit of work to figure out the right parameters to use to regenerate the client in the correct way, I added a script and config file to make this easier in the future.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
-# Grafeas
-An API to insert and retrieve annotations on cloud artifacts.
+# grafeas
+No description provided (generated by Swagger Codegen https://github.com/swagger-api/swagger-codegen)
```
Comment (wkozlik) [inline README.md:2]: Would it be okay to keep the original description?

File: config.py.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+    "packageVersion": "v1alpha1",
```
Comment (wkozlik) [inline config.py.json:2]: Would it make sense to use the same versioning scheme as in https://github.com/grafeas/client-go/blob/master/config.go.json ?

**Review by wkozlik — COMMENTED** [review]
Thank you for fixing this.

File: README.md (hunk 1 above)
Comment (nyc) [reply README.md]: Done, thanks for catching that.

File: config.py.json (hunk 2 above)
Comment (nyc) [reply config.py.json:2]: I believe that the go client is generating version `0.1.0` from the v1beta1 protos. I believe the package version for the client generated from the v1alpha1 protos are still versioned this way.

Example: https://github.com/grafeas/client-go/blob/master/v1alpha1/README.md

Comment (wkozlik) [reply config.py.json:3]: Thank you for the explanation.

**Review by wkozlik — APPROVED** [review]

**Pull request merged**